README.md

```diff
@@ -136,6 +136,7 @@ type ATable struct {
     Field2     int    `db:"field2"`                      // Use "db" in field tag to set column name used in SQL.
     Field3     int64  `db:"field3" fieldtag:"foo,bar"`   // Set fieldtag to a field. We can use methods like `Struct#SelectForTag` to use it.
     Field4     int64  `db:"field4" fieldtag:"foo"`       // If we use `s.SelectForTag(table, "foo")`, columnes of SELECT are field3 and field4.
+    Field5As  string `db:"field5" fieldas:"FIELD5"`      // Use "fieldas" in field tag to set AS name used in SELECT.
```

Comment:
```suggestion
    Field5As   string `db:"field5" fieldas:"FIELD5"`     // Use "fieldas" in field tag to set AS name used in SELECT.
```

Reply:
Is there a better description than "AS name" that we could use? Might help users searching for this feature in your library. I'm not sure if there is an official term, but I can see people using "alias" or "AS clause".

Could do something like:

> // Use "fieldas" in field tag to set a column alias/clause (AS) used in SELECT.

Or:

> // Use "fieldas" in field tag to set the returned column name (AS) using in SELECT.

Reply:
Updated.